browser/src/dom/NotebookbarAccessibility.js

```diff
@@ -272,6 +272,12 @@ var NotebookbarAccessibility = function() {
 		}
 	};
 
+	this.onInputKeyDown = function(event) {
+		if (event.ctrlKey) {
+			this.resetState();
```

Comment:
Why do we need this and also the event binding?

Reply:
For cases like Accessibility keys are activated ( Yellow boxes)

- then if use tries to save document ( CTRL + S ) or Find ( CTRL + F) 
- it works fine but there will a view JUMP from Home Tab to other Tab in notebookbar
- And the reason is we disable Accessibility on KeyUp so for some special keys like CTRL we should disbale the state on keyDown because in some cases other operations will execute before our accessibility fuinctions get disabled 

I have made a separate commit on this as well 

https://github.com/CollaboraOnline/online/pull/9999/commits/c3ff6cf5c64a6961f2270660fdb52707fd52ffff

Reply:
Thanks for the details :)